src/Roundabout/Roundabout.tsx

```diff
@@ -14,7 +13,7 @@ type RoundaboutType = {
     index: number;
     complete: boolean;
```

Comment:
Should partial and complete be mandatory? If so, why are they optionals in the getLabel function?